app/assets/stylesheets/farmbot.css.scss

```diff
@@ -107,20 +113,60 @@ $brown:          #CA8;
   color: $white;
   font-size: 20px;
   text-align: center;
-
+  display: inline-block;
+  font-style: normal;
+  cursor: pointer;
   &:after {
     content: '\f067';
   }
 }
+
+  .farm-designer-body {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .farm-designer-left {
+    order: -1;
+    background-color: $light_green;
+  }
+  .farm-designer-body {
+    flex-direction: row;
+    flex: 1;
+  }
+  .farm-designer-middle {
+    flex: 1;
+    background-color: $brown;
+  }
+  .farm-designer-left, .farm-designer-right {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Each selector in a comma sequence should be on its own line